test/todo/json5.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const tap = require('tap');
+const jstp = require('../..');
+
+const supportedByUs = {
```

Comment:
We should add noctal number tests here, because we decided before that we support them.